lib/kino_bumblebee/task_cell.ex

```diff
@@ -544,6 +631,57 @@ defmodule KinoBumblebee.TaskCell do
     ]
   end
 
+  defp to_quoted(%{"task_id" => "fill_mask"} = attrs) do
+    opts =
+      if(top_k = attrs["top_k"],
+        do: [top_k: top_k],
+        else: []
+      ) ++
+        [compile: [batch_size: 1, sequence_length: attrs["sequence_length"]]] ++
+        maybe_defn_options(attrs)
+
+    %{generation: generation} = variant_from_attrs(attrs)
+
+    [
+      quote do
+        {:ok, model_info} =
+          Bumblebee.load_model({:hf, unquote(generation.model_repo_id)}, log_params_diff: false)
+
+        {:ok, tokenizer} = Bumblebee.load_tokenizer({:hf, unquote(generation.tokenizer_repo_id)})
+
+        serving = Bumblebee.Text.fill_mask(model_info, tokenizer, unquote(opts))
+      end,
+      quote do
+        text_input = Kino.Input.textarea("Text", default: unquote(generation.default_text))
+        form = Kino.Control.form([text: text_input], submit: "Run")
+        frame = Kino.Frame.new()
+
+        form
+        |> Kino.Control.stream()
+        |> Kino.listen(fn %{data: %{text: text}} ->
+          one_mask? = match?([_, _], String.split(text, "[MASK]"))
```

Comment:
Alternatively we could do `try-rescue`, but by validating we have more control over the error message, and also don't promote `try-rescue` :)

Reply:
This is perfect to me. We could also just let it crash.

Reply:
We don't want to, because then the form would no longer be responsive!